api_v3/lib/types/enums/KalturaPartnerAuthenticationType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * @package api
+ * @subpackage enum
+ */
+class KalturaPartnerAuthenticationType extends KalturaEnum

```

Comment:
Need to define interface class PartnerAuthenticationType in core with these enum values , add implement to it here , and use the core enum in the core.
Avoid using this enum in places like kuserpeer